src/app/static/src/app/components/plots/choropleth/Choropleth.vue

```diff
@@ -3,13 +3,13 @@
         <filters class="col-md-3" v-if="includeFilters"
                  :filters="filters"
                  :selectedFilterOptions="selections.selectedFilterOptions"
-                 @update="onFilterSelectionsChange"></filters>
+                 @update:filters="onFilterSelectionsChange"></filters>
         <div id="chart" :class="includeFilters ? 'col-md-9' : 'col-md-12'">
-            <l-map ref="map" style="height: 800px; width: 100%" @ready="updateBounds" @vnode-updated="updateBounds">
+            <l-map ref="map" style="height: 800px; width: 100%" @ready="updateBounds">
```

Comment:
I have checked the styling in the map, we use a mock import for the map in the test so cannot check the @ready bit of the line but we shouldn't have to as long as we test updateBounds since we do not trigger the @ready event